src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpWinForms.cs

```diff
@@ -112,6 +119,8 @@ public void RenameControl()
             var project = new ProjectUtils.Project(ProjectName);
             VisualStudio.SolutionExplorer.OpenFileWithDesigner(project, "Form1.cs");
             VisualStudio.Editor.AddWinFormButton("SomeButton");
+            VisualStudio.SolutionExplorer.SaveFile(project, "Form1.cs");
+            VisualStudio.SolutionExplorer.SaveFile(project, "Form1.resx");
```

Comment:
This test is skipped and linked to an already-fixed issue. Could you unskip?

Reply:
Sure